doc/man3/OSSL_PARAM_BLD.pod

```diff
@@ -143,37 +141,39 @@ For both, the predefined key variables are:
 This example shows how to create an OSSL_PARAM array that contains an RSA
 private key.
 
-    OSSL_PARAM_BLD bld;
+    OSSL_PARAM_BLD *bld = OSSL_PARAM_BLD_new();
```

Comment:
Nit: This assignment should happen after variable declarations.

Well, maybe, this is just a code example..